test/integration-tests/resources/scenario-non-local-templates/0-tasks.yml

```diff
@@ -0,0 +1,19 @@
+
+OrganizationUpdate:
+  Type: update-organization
+  Skip: true
+  Template: ./organization.yml
+
+CopyToS3:
+  Type: copy-to-s3
+  RemotePath: s3://org-formation-integration-test/xyz/buckets.yml
+  LocalPath: ./buckets.yml
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Region: eu-central-1
+
+MyRoles:
+  DependsOn: CopyToS3
+  Type: update-stacks
+  StackName: buckets-from-remote-template
+  Template: s3://org-formation-integration-test/xyz/buckets.yml #todo: support !Sub
```

Comment:
the commit message seemed to imply that `!sub` is supported in this PR?

Reply:
True, that Sub needs to be replaced before the file is read, which might make it a bit tricky. Wanted to first load a file from s3, https, then look into this and documentation.